xdsl/dialects/emitc.py

```diff
@@ -50,6 +62,97 @@
 from xdsl.utils.exceptions import VerifyException
 from xdsl.utils.hints import isa
 
+EmitCIntegerTypeBitwidthConstr = functools.reduce(
+    operator.or_, (EqAttrConstraint(bw) for bw in (i1, i8, i16, i32, i64))
```

Comment:
```suggestion
    operator.or_, (EqAttrConstraint(bw) for bw in (1, 8, 16, 32, 64))
```

Reply:
Ah my bad I just realised this isn't expressible yet I'll add a new constraint quickly.

Reply:
#5040